modules/runtime/pages/purge-tool.adoc

```diff
@@ -164,29 +179,41 @@ NOTE: The `--before-date` parameter must be in https://www.epochconverter.com/[m
 
 ==== Delete modes
 
-===== `batch-delete`
+The `delete` command supports two modes: `batch-delete` and `copy-truncate`.
 
-By default, the tool uses the `batch-delete` mode to delete rows in database tables. +
-This mode will be slower than the copy-truncate mode, but it doesn't require the Runtime to be shutdown and can be stopped and resumed at any time.
+===== `batch-delete`
 
-In this mode, each deleted batch is committed in database. +
-The batch size and the timeout interval between each batch can be configured using the `--batch-size` and `--timeout-interval` options on the `delete` command.
+The default mode is `batch-delete`. It can be used while the Bonita runtime is still running, and can be stopped and resumed at any time. +
+We recommend to use this mode to delete small to medium-sized data volumes regularly, for example, by using cron jobs during off-peak hours.
 
-Fine tune the batch size and timeout interval depending on your database configuration and the volume of data to delete. By default, the batch size is `5000` and the timeout interval is `0` ms.
+In this mode, the rows of each table to purge are deleted in batches, and each batch is committed in the database.
 
 ===== `copy-truncate`
 
-In this mode, the tool will copy the rows to keep in a temporary table, then truncate and drop the original table and rename the temporary table and recreate all required constraints and indices.
+The `copy-truncate` mode is more efficient for large data volumes, especially when the number of rows to delete is higher than the number of rows to keep in the tables, but it requires the Bonita runtime to be stopped during the operation. Also, we advise not to stop the ongoing execution.
+
+In this mode, the rows to keep are copied in batches to a temporary table, then the original table is truncated and dropped, and the temporary table is renamed to the original table. All required constraints and indexes are re-created.
 
-Due to the efficiency of the TRUNCATE command, this method should be faster in most cases, especially when the number of rows to delete is higher than the number of rows to keep in the table, but it requires the Runtime to be stopped during the operation.
+==== Fine-tuning the deletion
 
-Use the `--delete-mode copy-truncate` option to use this mode with the `delete` command.
+As mentioned before, both modes use batch processing to either delete or copy rows. +
+The batch processing can be fine-tuned using the `--batch-size` and `--batch-interval` options:
+
+    * `--batch-size` to define the number of rows to delete or copy in each batch. +
+With the `batch-delete` mode, although it depends on your database configuration and the volume of data to delete, we recommend starting with a batch size of `5000` rows. You can increase this value based on the performance of your database. +
+With the `copy-truncate` mode, you can use a larger batch size to improve performance.
```

Comment:
How much larger? We should indicate a value, or a formula to calculate the value.